aws-aurora-postgres/variables.tf

```diff
@@ -112,7 +112,7 @@ variable "kms_key_id" {
 variable "engine_version" {
   type        = string
   description = "The version of Postgres to use. This should be a *prefix* if auto version upgrades are enabled. (Docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#engine_version)"
-  default     = "10"
+  default     = "15.2"
```

Comment:
This could be a breaking change... Isn't it safer to upgrade one major version at a time?